client/src/components/Connections/dataWrappers/EntitySearchWrapper.js

```diff
@@ -17,6 +18,7 @@ const EntitySearchWrapper = (WrappedComponent: ComponentType<*>) => {
         {...props}
         entitySearch1={query.eid1 || ''}
         entitySearch2={query.eid2 || ''}
+        showGraph={query.graph === 'true' || false}
```

Comment:
Why is that `|| false` here? Can `===` return another falsy value?

Reply:
:man_facepalming: